templates/layouts/backend.tpl

```diff
@@ -86,19 +86,22 @@
 					<dropdown class="app__headerAction app__userNav">
 						<template #button>
 							<initials-avatar
+								initials="{$currentUser->getDisplayInitials()}"
 								:is-secondary="true"
 								{if $isUserLoggedInAs}
 								:is-disabled="true"
 								{/if}
 							></initials-avatar>
+							<span class="-screenReader">{$currentUser->getData('userName')}</span>
 							{if $isUserLoggedInAs}
 								<initials-avatar
-									class="absolute right-2 top-2 rounded-full h-5 w-5"
+									initials="{$originalUser->getDisplayInitials()}"
+									class="absolute right-2 top-0 rounded-full h-5 w-5"
 									:is-warnable="true"
 									:shrink="true"
 								></initials-avatar>
+								<span class="-screenReader">{$originalUser->getData('userName')}</span>
```

Comment:
Should be passed through `|escape` modifier to avoid XSS problems:
```
<span class="-screenReader">{$originalUser->getData('userName')|escape}</span>
```